__tests__/cmds/changelogs/index.test.ts

```diff
@@ -37,21 +37,23 @@ describe('rdme changelogs', () => {
     delete process.env.TEST_CI;
   });
 
-  it('should error if no folder provided', () => {
-    return expect(changelogs.run({ key })).rejects.toThrow(
-      'No folder provided. Usage `rdme changelogs <folder> [options]`.'
+  it('should error if no path provided', () => {
+    return expect(changelogs.run({ key })).rejects.toStrictEqual(
```

Comment:
Why not do `.toThrow` here anymore?

Reply:
`.toThrow` is technically a partial match on `err.message`, `.toStrictEqual(<full error object>)` is a stricter check